userguide/snippets/es12.rst

```diff
@@ -219,3 +219,9 @@ the bezel by aligning it with the pins on the bezel ears and pressing
 it into place.
 
 
+.. _User Guide:
+
+User Guide
+~~~~~~~~~~
```

Comment:
Why change the heading level here?  This will put the Guide section as part of the current chapter, rather than a new chapter like it was.